packages/atomic/src/components/facets-v1/facet-search/facet-search-utils.ts

```diff
@@ -37,3 +39,20 @@ export function shouldDisplaySearchResults(facetSearchState: FacetSearchState) {
 
   return !isLoading;
 }
+
+export function highlightSearchResult(
+  resultValue: string,
+  searchQuery?: string
+) {
+  const sanitizedResult = escape(resultValue);
+
+  if (!searchQuery || searchQuery.trim() === '') {
```

Comment:
I'd switch to a strict null or undefined check, or set default parameter value in the function to empty string.

Reply:
hey hey this was a draft! 😄